<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventWindow</class>
 <widget class="QWidget" name="EventWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>EventGift</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QWidget" name="gifts" native="true">
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayoutGifts">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="leaveEvent">
            <property name="text">
             <string>Leave Event</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollAreaGifts">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContentsGifts">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>230</width>
             <height>500</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="giftListLayout"/>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectGift">
          <property name="text">
           <string>Select Gift</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="info" native="true">
       <layout class="QVBoxLayout" name="verticalLayoutInfo">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="eventTitle">
            <property name="text">
             <string>Event title:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="eventTitleLabel">
            <property name="text">
             <string>eventTitleLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="place">
            <property name="text">
             <string>Place:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="placeLabel">
            <property name="text">
             <string>placeLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="date">
            <property name="text">
             <string>Date:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="dateLabel">
            <property name="text">
             <string>dateLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="time">
            <property name="text">
             <string>Time:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="timeLabel">
            <property name="text">
             <string>timeLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="describeEdit"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="users" native="true">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayoutUsers">
        <item>
         <widget class="QScrollArea" name="scrollAreaUsers">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContentsUsers">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>180</width>
             <height>560</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="userListLayout"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
